src/components/Button.js

```diff
@@ -60,7 +61,8 @@ export const Button = ({
         needLandscapeStyle && styles.titleLandscape,
         big && styles.bigTitle,
         small && styles.smallTitle,
-        smallest && styles.smallestTitle
+        smallest && styles.smallestTitle,
+        invert && lightest && styles.titleLightestInvert
```

Comment:
why checking `invert` again here? `invert` and `lightest` should be seen independent of each other. if we use both props, both will be applied. so `lightest` should work like all the others with `lightestTitle` here.

Reply:
I made a new force-commit and applied the changes you mentioned with this commit.